Update Docker Compose docs, refs #13119

jraddaoui opened the pull request:
Some minor modifications are needed due to the changes made in the
environment since the creation of this page.

Fixes #54.

fiver-watson commented on dev-manual/env/compose.rst:
```diff
@@ -75,6 +74,22 @@ don't want to do so each time we invoke the ``docker-compose`` command.
+
+   In some operating systems, the default virtual memory limits can be low.
+   Elasticsearch increases one of those limits when installed via packages and
+   in the container that will be created bellow. However, the same limit needs
```
One minor typo here - it's below, not bellow :)

fiver-watson reviewed (changes requested):
Only one minor typo noted - otherwise it looks good!

jraddaoui committed:
Update Docker Compose docs, refs #13119

Some minor modifications are needed due to the changes made in the
environment since the creation of this page.

fiver-watson reviewed (approved):
👍 LGTM! Thanks!
